sphinx/builders/linkcheck.py

```diff
@@ -107,13 +112,25 @@ def init(self) -> None:
     def check_thread(self) -> None:
         kwargs = {
             'allow_redirects': True,
-            'headers': {
-                'Accept': 'text/html,application/xhtml+xml;q=0.9,*/*;q=0.8',
-            },
-        }
+        }  # type: Dict
         if self.app.config.linkcheck_timeout:
             kwargs['timeout'] = self.app.config.linkcheck_timeout
 
+        def get_request_headers() -> Dict:
+            url = urlparse(uri)
+            candidates = ["%s://%s" % (url.scheme, url.netloc),
+                          "%s://%s/" % (url.scheme, url.netloc),
```

Comment:
@tk0miya why is this entry duplicated?

Reply:
Oh, I see. There's a trailing slash.

Reply:
Yes, it's intended. You can use either one in your setting.